oschecks/check/check_gnocchi.py

```diff
@@ -0,0 +1,79 @@
+# -*- coding: utf-8 -*-
+
+from gnocchiclient import client as gnocchi_client
+from gnocchiclient import exceptions as gnocchi_exceptions
+
+import oschecks.openstack as openstack
+import oschecks.common as common
+
+
+class GnocchiCommand(openstack.OpenstackCommand):
+    '''Base class for Gnocchi checks.'''
+
+    def get_parser(self, prog_name):
+        parser = super(GnocchiCommand, self).get_parser(prog_name)
+        if not hasattr(self, 'arg_group'):
```

Comment:
So the thing is that GnocchiCommand is a base class for CheckResourceTypeExists. The conditional is there in order to not create the same argument group twice. I must admit, that I did not tested(/read docs) if parser.add_argument_group() with the same caption will return the previously created group or will create the same group again. Since I always count with the worse option, I assumed I had to take care of the unwanted behaviour.